src/app/component/MyAccount/MyAccount.style.scss

```diff
@@ -295,4 +295,25 @@
         bottom: 0;
         margin: auto;
     }
+
+    section {
+        button {
+            background-color: var(--color-myaccount-transparent);
```

Comment:
No, I mean, there are --button-color etc... vraiables. Could you manage the styles via them?